components/UI/Drawer/components/channels/babala/FeedChannelBabala.tsx

```diff
@@ -20,13 +20,17 @@ const FeedChannelBabala = ({
       }
     });
 
+  const isReasonDisplayable = !!reason && reason !== "NaN";
```

Comment:
this does not seem reactive. Can you hold this variable with useState and make it reactive with useEffect?

Reply:
Reason yukarıdan props olarak geldiği için bunu state'de tutmaya gerek olmadığını düşünüyorum. Hali hazırda string bir ifade olduğundan farklı bir değer sonrası component re-render olacaktır. Tabi yanlış da değerlendirmiş olabilirim kaynak:

https://beta.reactjs.org/learn/you-might-not-need-an-effect

Reply:
Ufak bir oneri, en yukarida `isNaNstring` diye bir method tanimlayip hem 14. satirda hem burada kullanabilir miyiz? Belli ki `nan` da gelebiliyor, `NaN` da gelebiliyor.

Reply:
@eytanfb Could you please update this.